packages/adapter/src/lib/adapter/adapter.ts

```diff
@@ -11888,63 +11895,61 @@ export class AdapterClass extends EventEmitter {
             });
         }
 
-        return new Promise(resolve => {
-            this._extendObjects(objs, resolve);
-        });
-    }
-
-    private async _extendObjects(tasks: Record<string, any>, callback: () => void): Promise<void> {
-        if (!tasks || !tasks.length) {
-            return tools.maybeCallback(callback);
-        }
-        const task = tasks.shift();
-        const state = task.state;
-        if (state !== undefined) {
-            delete task.state;
-        }
-
         try {
-            tools.validateGeneralObjectProperties(task, true);
-        } catch (e) {
-            this._logger.error(`${this.namespaceLog} Object ${task._id} is invalid: ${e.message}`);
-            return tools.maybeCallbackWithError(callback, e);
-        }
-
-        if (!this.#objects) {
-            this._logger.info(
-                `${this.namespaceLog} extendObjects not processed because Objects database not connected.`
+            await this._extendObjects(objs);
+        } catch (err) {
```

Comment:
Errors in catch are always named `e` in the codebase or changed towards it on changes

Reply:
Renamed